app/views/helpers/Formatters.scala

```diff
@@ -68,8 +68,8 @@ trait CurrencyFormatters {
 
 trait FileFormatters {
 
-  val kbThreshold = 1000
-  val mbThreshold = 1000000
+  private val kbThreshold = 1024
```

Comment:
Could you please write tests for these two threshold updates for the fileSize method?